packages/flutter/test/gestures/monodrag_test.dart

```diff
@@ -45,8 +45,14 @@ void main() {
     GestureBinding.instance.gestureArena.close(91);
     tester.async.flushMicrotasks();
 
-    GestureBinding.instance.handleEvent(up90, HitTestEntry(MockHitTestTarget()));
-    GestureBinding.instance.handleEvent(up91, HitTestEntry(MockHitTestTarget()));
+
+    // Regression test for https://github.com/flutter/flutter/issues/112403.
+    v.checkStart(null, down91.pointer);
```

Comment:
Hm, I believe this isn't a full regression test. If someone where to accidentally add the exclamation mark back to line 366 in monodrag.dart (the root cause for the crash), this test would continue to pass?

Reply:
Just calling `acceptGesture` twice in a row would trigger this bug.

There seem to be several ways that the last event can get nulled out and not reset depending on which sequence of pointer events get submitted. I don't know which of those would be expected or not and which we should have better test coverage of.